apiary.apib

```diff
@@ -105,6 +105,11 @@ Get the key for region
 
 Upload the keys with ssh and gpg format. 
 
+* The operation need to have a valid token to process it. To obtain
+the token, you need to have an admin account in FIWARE Lab
+(account.lab.fi-ware.org). With the credentials (username, password
```

Comment:
Once that you have these credentias (user name, password and tenant name), you can obtain the token identifier (access.token,id path) in the response message in json format to the Keystone service.